effects/effect.py

```diff
@@ -14,8 +14,9 @@ class EffectStatus(Enum):
 
 
 class Effect(ABC):
-    def __init__(self, game: Game, name: str, creator: Citizen,
-                 priority: int) -> None:
+    def __init__(self, context: 'Context', game: Game, name: str,
+                 creator: Citizen, priority: int) -> None:
+        self.context = context
         self.game = game
```

Comment:
you shouldn't store game inside of effect.
Game should be fetched from Context

Reply:
Fixed